Client/metrics.yaml

```diff
@@ -140,6 +140,30 @@ accessibility:
     expires: "2023-06-01"
 # App menu
 app_menu:
+  homepage_menu:
+    type: counter
+    description: |
+        Counts the number of times a user taps in the menu toolbar
+        on the homepage
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-5442
```

Comment:
Can we link the github tickes instead of jira please https://github.com/mozilla-mobile/firefox-ios/issues/12697